index.css

```
@@ -0,0 +1,63 @@
+body {
+  margin: 0;
+  font-family: Arial, sans-serif;
+}
+
+header {
+  background-color:  #0088F080;
+  color: white;
+  padding: 20px;
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo img {
+  width: 190px;
+  height: 40px;
+  top: 7px;
+  left: 210px;
+  padding: 0px 14px 0px 0px;
+  gap: 0px;
+  opacity: 0px;
+  
+}
+
+.search-bar {
+  display: flex;
+  align-items: center;
+}
+
+.search-bar input {
```

Es mejor crear una clase que aplicar el estilo al elemento
```suggestion
.search-bar__input {
```